python/lsst/ip/diffim/modelPsfMatch.py

```diff
@@ -300,6 +300,7 @@ def run(self, exposure, referencePsfModel, kernelSum=1.0):
         psfMatchedExposure = afwImage.ExposureF(exposure.getBBox(), exposure.getWcs())
         psfMatchedExposure.setFilter(exposure.getFilter())
         psfMatchedExposure.setCalib(exposure.getCalib())
+        psfMatchedExposure.setPsf(referencePsfModel)
```

Comment:
It doesn't still need to be there.  Because it has only been there for 4 months, it is *extremely* unlikely that anyone's non-stack scripts depend on it now, so I'm comfortable making the interface change of removing it.